src/Parser/Warrior/Arms/CONFIG.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import SPECS from 'common/SPECS';
+import SPEC_ANALYSIS_COMPLETENESS from 'common/SPEC_ANALYSIS_COMPLETENESS';
+
+import CombatLogParser from './CombatLogParser';
+import CHANGELOG from './CHANGELOG';
+
+import thebadbossy_avatar from './Images/thebadbossy_avatar.jpg';
+
+export default {
+  spec: SPECS.ARMS_WARRIOR,
+  // TODO: Make maintainer an array
+  maintainer: '@TheBadBossy',
+  maintainerAvatar: thebadbossy_avatar,
+  description: (
+    <div>
+      Hey I've been hard at work making this analyzer for you. I hope the suggestions give you useful pointers to improve your performance. Remember: focus on improving only one or two important things at a time. Improving isn't easy and will need your full focus until it becomes second nature to you.<br /><br />
+      </div>
+  ),
+  // When changing this please make a PR with ONLY this value changed, we will do a review of your analysis to find out of it is complete enough.
+  completeness: SPEC_ANALYSIS_COMPLETENESS.GREAT,
+  specDiscussionUrl: 'https://github.com/WoWAnalyzer/WoWAnalyzer/issues/54',
```

Comment:
Both line 22 and 23 just need to be updated with the spec specific information.

`SPEC_ANALYSIS_COMPLETENESS.NEEDS_MORE_WORK` should be used for now.